### PR TITLE
GUI/Core/Parse Status/185

### PR DESCRIPTION
**Pull request by GuyomT**
parsing the status response from the server and sending it to the map

### CODE REVIEW

**Review by LucasTesnier — APPROVED** [review]
NTR!

File: src/zappy_gui_src/Core/Entity/Entity.hpp [hunk 1]
```diff
@@ -120,6 +120,24 @@ namespace gui::entity {
...
     };
+
+    struct Status {
+        public:
+        /// \brief The status of the game win or loose
+        std::string _status;
+        /// \brief the team who won or lost
+        std::string _team_name;
+        /// \brief get the team name
+        /// \return a string containing the name of the team
```
Comment (osvegn) [inline src/zappy_gui_src/Core/Entity/Entity.hpp:131]: invalid doc

**Review by osvegn — CHANGES_REQUESTED** [review]
Update the doc and it will be ok

**Review by pariciadra — APPROVED** [review]
Lgtm

**Review by osvegn — APPROVED** [review]

**Pull request merged**